onward/app/views/fragments/videosInSeries.scala.html

```diff
@@ -1,4 +1,4 @@
-@(videos: List[model.Video], seriesTitle: Option[String])(implicit request: RequestHeader)
+@(videos: List[model.Video], seriesTitle: Option[String], series: String, nextPage: Option[Int], prevPage: Option[Int])(implicit request: RequestHeader)
```

Comment:
Keeping it consistent with the naming elsewhere - this will be refactored once in.